src/app/review-item/components/review-page/review-page.component.ts

```diff
@@ -49,23 +52,23 @@ export class ReviewPageComponent implements OnInit, UnsavedChanges {
 
   ) {
     this.showQuestionaire = false;
-    this.questions = [
-      {
-        title: "Woran erkenne ich eine gute Quelle?",
-        description: "Hier haben wir alles zusammengefasst um dir zu helfen gute Quellen zu erkennen",
-        icon: "thing"
-      },
-      {
-        title: "Woran erkenne ich eine gute Quelle?",
-        description: "Hier haben wir alles zusammengefasst um dir zu helfen gute Quellen zu erkennen",
-        icon: "thing"
-      },
-      {
-        title: "Woran erkenne ich eine gute Quelle?",
-        description: "Hier haben wir alles zusammengefasst um dir zu helfen gute Quellen zu erkennen",
-        icon: "thing"
-      },
-    ]
+    // this.questions = [
```

Comment:
this.questions is already used for another component.

```    <div class="questions-section--list">
      <div class="question" *ngFor="let q of questions; let i=index">
          <div class="question--icon">{{q.icon}}</div>
          <div class="question--title">{{q.title}}</div>
          <div class="question--description">{{q.description}}</div>
      </div>
    </div>
   ```
   
   Could you put it back?
   You could rename the old variable, and make sure it still works.
   
  Or you could choose a new name for the new variable.